README.md

```diff
@@ -29,17 +29,17 @@ The plugin can be configured in the [**semantic-release** configuration file](ht
 
 ```json
 {
-  "plugins": [
-    "@semantic-release/commit-analyzer",
-    "@semantic-release/release-notes-generator",
-    [
-      "semantic-release-slack-bot",
-      {
-        "notifyOnSuccess": false,
-        "notifyOnFail": true
-      }
-    ]
-  ]
+	"plugins": [
+		"@semantic-release/commit-analyzer",
+		"@semantic-release/release-notes-generator",
+		[
+			"semantic-release-slack-bot",
+			{
+				"notifyOnSuccess": false,
+				"notifyOnFail": true
+			}
+		]
+	]
```

Comment:
I think an 8 space indentation looks strange in a markdown file. Semantic-release uses 2 spaces in their documentation so I think we should try to align to them.

Reply:
Done

Reply:
Now there are tabs. I looked at the prettier config and it specifies tabs, but we should use 2 spaces. That is another issue that I will look into.

Could you remove this chunk from the PR, and I will update the prettier config in another PR.